client/Assets/Scripts/Runtime/Beamable/Samples/Core/Utilities/ExampleProjectHacks.cs

```diff
@@ -22,10 +24,8 @@ public static class ExampleProjectHacks
         /// </summary>
         public static void ClearDeviceUsersAndReloadScene()
         {
-            // Reset the system. Then reload the current scene
-            PlatformService platformService = ServiceManager.Resolve<PlatformService>();
-            platformService.ClearDeviceUsers();
-            ServiceManager.OnTeardown();
+            Beam.ClearAndStopAllContexts()
+                .FlatMap(_ => Beam.ResetToScene(SceneManager.GetSceneByBuildIndex(0).name));
```

Comment:
Why not `Then` instead of `FlatMap`?

Reply:
Well, it's just copy-pasted `reset` command from our Admin Console. The outcome of `FlatMap` and `Then` is exactly the same in this context.